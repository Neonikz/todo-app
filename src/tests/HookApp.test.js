import React from 'react'
import { HooksApp } from "../HookApp"
import {shallow} from 'enzyme';

describe('Pruebas en el HookApp', () => {
    test('Hacer el primer snapshot', () => {
        const wrapper = shallow(<HooksApp />);
        expect(wrapper).toMatchSnapshot();
    })
    
})
