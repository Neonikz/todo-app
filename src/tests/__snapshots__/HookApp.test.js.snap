// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el HookApp Hacer el primer snapshot 1`] = `
<Fragment>
  <h1>
    Hola mundo
  </h1>
</Fragment>
`;
